qemu/tests/hugepage_mem_stress.py

```diff
@@ -0,0 +1,73 @@
+import time
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_test
+from virttest.utils_test import BackgroundTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu hugepage memory stress test.
+    Steps:
+    1) System setup hugepages on host.
+    2) Mount this hugepage to /mnt/kvm_hugepage.
+    3) Run memory heavy stress inside guest.
+    4) Check guest call trace in dmesg log.
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    def heavyload_install():
+        if session.cmd_status(test_installed_cmd) != 0:
+            logging.warning("Could not find installed heavyload in guest, will"
+                            " install it via winutils.iso ")
+            winutil_drive = utils_misc.get_winutils_vol(session)
+            if not winutil_drive:
+                test.cancel("WIN_UTILS CDROM not found.")
+            install_cmd = params["install_cmd"] % winutil_drive
+            session.cmd(install_cmd)
+
+    os_type = params["os_type"]
+    stress_duration = params.get_numeric("stress_duration", 180)
```

Comment:
So what does mean of `stress_duration`, you have already defined `--timeout 30s` in the stress arguments,

Reply:
Corrected